packages/documentation/src/stories/components/tooltip/tooltip.docs.mdx

```diff
@@ -0,0 +1,30 @@
+import { Canvas, Controls, Meta } from '@storybook/blocks';
+import * as TooltipStories from './tooltip.stories';
+import LinkTo from '@storybook/addon-links/react';
+
+<Meta of={TooltipStories} />
+
+# Tooltip
+
+<div className="lead">For short, informative messages or explanations.</div>
+
+<Canvas of={TooltipStories.Default} />
+<Controls of={TooltipStories.Default} />
+
+## Installation
```

Comment:
Thanks for moving the section, can you update the title structure to: 

- Installation
- Examples
  - Tooltip on non-focusable elements
  - One tooltip on multiple elements